README.md

```diff
@@ -1,3 +1,107 @@
 # safe-react-components
 
-[Live example](https://components.gnosis-safe.io/)
\ No newline at end of file
+![Badge](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg) 
+![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/gnosis/safe-react-components?sort=semver)
+![GitHub](https://img.shields.io/github/license/gnosis/safe-react-components)
+
+
+This repository contains a set of React components written in typescript.
+
+These components are being used to build the [Safe Multisig](https://github.com/gnosis/safe-react) web and desktop app.
+
+As Safe Multisig allows to integrate third party applications ("Safe Apps"), these components can also be used to build Safe Apps with the following benefits:
+
+- **Native feel:** Build your Safe Apps with a similar style as the one used Safe-multisig. This makes your Safe Apps feel almost like a native feature of the Safe Multisig.
+- **Responsive:** Most of the components will are optimized to render properly in different resolutions.
+- **Blockchain-focused:** Some components solve common blockchain-related problems like inputs for ETH addresses and bigNumbers, identicon images, and more.
+- **Save time:** No need to build all components from scratch.
+
+
+## How to install
+
+```bash
+   yarn add @gnosis.pm/safe-react-components
+```
+
+## Integration
+
+This library makes use of [material-ui](https://material-ui.com/) as a `peerDependency`, this means you must install it in your Safe App. Make sure to provide the same version as the one being used by the current version of this library.
+
+Once everything is installed, you have to instantiate a [ThemeProvider](https://styled-components.com/docs/api#themeprovider) from [styled-components](https://@gnosis.pm/safe-react-components/).
```

Comment:
why does `styled-components` link to `https://@gnosis.pm/safe-react-components/`